apps/opik-frontend/src/router.tsx

```diff
@@ -286,32 +295,6 @@ const compareTrialsRoute = createRoute({
   },
 });
 
-// Optimization Studio
-const optimizationStudioRoute = createRoute({
-  path: "/optimization-studio",
```

Comment:
Those that navigated to old link, will simply need to re-navigate to the new 1? No auto redirect?

If it's complicated or in-correct, let's not worry about this 😄